doc_src/language.rst

```diff
@@ -516,7 +516,7 @@ Without the quotes or braces, fish will try to expand a variable called ``$WORDs
 
 The latter syntax ``{$WORD}`` is a special case of :ref:`brace expansion <expand-brace>`.
 
-If $WORD here is undefined or an empty list, the "s" is not printed. However, it is printed if $WORD is the empty string (like after ``set WORD ""``).
+If $WORD is undefined or an empty list, the "s" is printed after the quotes expansion but not after the braces expansion. However, it is printed if $WORD is the empty string (like after ``set WORD ""``).
```

Comment:
> If $WORD is undefined or an empty list, the "s" is printed after the quotes expansion but not after the braces expansion

This is misleading, because nothing will actually be printed due to [Cartesian product expansion](https://fishshell.com/docs/current/index.html#combining-lists-cartesian-product).

What about just dropping this line instead? It describes what happens for some cases but the reasons are only given in the referenced sections (_expand-brace and _cartesian-product).

Reply:
I agree, it would be simpler to leave that particular effect to the relevant section, Cartesian product expansion, as it may over complicate the current topic, variable expansion.

Can I also ask about:

> The latter syntax ``{$WORD}`` is a special case of :ref:brace expansion <expand-brace>.

According to my, albeit rookie, understanding I was under the impression that {$WORD} is not a special case of brace expansion but just normal brace expansion with only was item.

Reply:
Fair enough.